pomegranate/gmm.pyx

```diff
@@ -232,7 +288,7 @@ cdef class GeneralMixtureModel(BayesModel):
 		if not self.is_vl_:
 			X_ndarray = _check_input(X, self.keymap)
 			X_ptr = <double*> X_ndarray.data
-			
+
```

Comment:
This and many lines of difference below are an example of the noise eliminated by #273.